pkg/clustermesh/clustermesh.go

```diff
@@ -114,6 +117,7 @@ func NewClusterMesh(c Configuration) (*ClusterMesh, error) {
 			Name:      "remote_clusters",
 			Help:      "The total number of remote clusters meshed with the local cluster",
 		}, []string{metrics.LabelSourceCluster, metrics.LabelSourceNodeName}),
+		IPCache: ipcache.NewIPCache(),
```

Comment:
Creating a new ipcache here is a bit surprising, should this not be passed in as a parameter to `NewClusterMesh()` instead?

Reply:
Ah you're quite right, I got mixed up thinking that this was the clustermesh-apiserver binary. :pray: